schedule/yam/test_cases/yast_no_self_update_aarch64.yaml

```diff
@@ -5,12 +5,11 @@ description:    >
   hard checks are done that the self-updating is really disabled.
   Installation is validated by successful boot and that YaST does not report
   any issue.
-vars:
-  INSTALLER_NO_SELF_UPDATE: 1
-  YUI_REST_API: 1
 schedule:
   license_agreement:
     - installation/validate_no_self_update
     - installation/licensing/accept_license
+  product_selection:
+    - installation/validate_no_self_update
```

Comment:
You are replicating line#10 here, so your VR has run validate_no_self_update twice.
Same is true for the x86_64 schedule.
So if you omit those 2 lines you could easily use this file for both `aarch64` and `x86_64` because `install_SLES` is part of the `default_x86_64.yaml` already.

Reply:
yes, and anyway the rule is that in folder  `test_cases` we don't want scheduled referring to an specific architecture.

Reply:
Solved here 5e8a9ac6fa5f85b5c1c4c05ef1724c79a0ab7786

Thanks!